easycord/_bot_commands.py

```
@@ -99,6 +108,9 @@ def _build_slash_callback(
         ephemeral: bool = False,
         permissions: list[str] | None = None,
         cooldown: float | None = None,
+        nsfw: bool = False,
```

Ruff (0.15.10)

[warning] Unused method argument: `nsfw`

(ARG002)